instagram/src/components/PostContainer/PostsPage.js

```diff
@@ -0,0 +1,53 @@
+import React from "react"
+import PostContainer from '/Users/lauradaugherty/Lambda Projects/React-Insta-Clone/instagram/src/components/PostContainer/PostContainer.js'
+import SearchBar from '/Users/lauradaugherty/Lambda Projects/React-Insta-Clone/instagram/src/components/SearchBar/SearchBar.js'
+import dummyData from '/Users/lauradaugherty/Lambda Projects/React-Insta-Clone/instagram/src/dummy-data.js'
+
+class PostsPage extends React.Component {
+  constructor () {
+    super ();
+    this.state = {
+      datas: [],
+    }
+    // console.log('construction')
+  };
+
+  componentDidMount() {
+    this.setState ({
+      datas: dummyData,
+    })
+    // console.log('didMount')
+  }
+  
+  //function to filter datas with an argument
+  filterData = (searchInput) => {
+    console.log(searchInput)
+    //props.datas.filter() data => this.prevState.searchInput === props.datas.username)
+    this.setState({
+      datas: dummyData.filter((data) => {
+        console.log("inside setSTATE")
+        console.log(data)
+        console.log(data.username)
+        return (
+          data.username.includes(searchInput)
```

Comment:
This works well but we can make it even better! Right now, your search works only if the casing is the same. We can make this more robust by editing the function so that if a user is using capitalization, this will still work. Change it to the following: 

data.username.toLowerCase().includes(searchInput.toLowerCase())